### ISSUE TITLE
[a11y]: multimodal support for post with alternative description

### ISSUE THREAD

**Issue by grittypuffy** [grittypuffy/civicsphere#38]
Currently, the files uploaded in community post do not support alt tags for images and video transcriptions or summarization, which impacts accessibility for screen reader users. This is due to lack of tagging of file metadata.

An approach that can work is including the file type in the BSON document to improve performance by reducing calls to blob storage, at the cost of higher database size.

**Comment by grittypuffy** [comment]
Been working on [TranscribeIt](https://github.com/fossiaorg/transcribeit) for video transcriptions for live streams and online content, it supports any online service that is supported by yt-dlp.

It will need more work to make it work with external object storage and local files, but possible.
